.github/workflows/pr.yaml

```diff
@@ -72,6 +72,23 @@ jobs:
       - name: Lint python
         run: yarn lint-python
 
+  lint-jinja:
+    runs-on: ubuntu-22.04
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Install node dependencies
+        run: yarn install --immutable
+
+      - name: Install python dependencies
+        run: |
+          python3 -m pip install --upgrade pip
+          sudo pip3 install djlint
+
+      - name: Lint jinja
+        run: yarn lint-jinja
```

Comment:
It seems that `lint-jinja` script only checks modified files (`git diff --name-only HEAD`), so would that even work on CI when everything is commit, there are no local changes?

Reply:
that's a good point, it's useful to have this script locally before you have committed changes. But after you have committed it's not going to work. My idea was to run this check in CI only for files from the PR, but not for the whole codebase. Because in the team we decided to apply linting step by step, only to files that are being changed.

Reply:
Found something like this: https://github.com/marketplace/actions/changed-files

Reply:
thanks, I also find this https://github.com/codeocelot/git-diff-lint/blob/master/git-diff-lint, so currently playing around and checking if that works for us

Reply:
@bartaz I ended up applying your finding and made it work nicely now. I will check all changed files inside the `templates` folder and lint them. You can check that it works by looking at the logs of the latest CI GH action below called `lint-jinja`. I changed one HTML file inside `templates` folder and you can see that it's being linted.